html/semantics/embedded-content/the-img-element/Image-constructor.html

```diff
@@ -28,10 +28,10 @@
 
   test(function() {
       assert_equals(Image.name, "Image", "Image name should be Image (not HTMLImageElement)");
-      assert_equals(Image.__proto__, Function.prototype, "Image __proto__ is Function prototype");
+      assert_equals(Object.getPrototypeOf(Image), Function.prototype, "Object.getPrototypeOf(Image) is Function prototype");
```

Comment:
```suggestion
      assert_equals(Object.getPrototypeOf(Image), Function.prototype, "Image's prototype is Function.prototype");
```

These assert descriptions weren't great before, and what "Function prototype" means isn't entirely clear. Suggestion consistent wording for all of these, but because of a weird GitHub multiline select issue I can't seem to do it in a single suggestion.